Mage/src/main/java/mage/abilities/effects/common/CreateTokenEffect.java

```diff
@@ -183,6 +194,39 @@ private void setText() {
             }
         }
 
+        for (int i = 1; i < tokens.size(); i++) {
```

Comment:
is it possible to condense this rather than handle the first separately with duplicate code?